extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php

```diff
@@ -115,6 +115,7 @@
 	'user-identity-remove-fail' => 'There was some problem when removing the avatar!',
 	'user-identity-avatars-maintenance' => 'Upload and deletion of avatars temporarily disabled during maintenance!',
 	'userprofilepage-edit-modal-error' => 'Unexpected error occurred',
+	'user-identity-token-mismatch' => 'Oops! Token doesn\'t match',
```

Comment:
What do you think about having this message in common place and reuse it everywhere?

BTW current message is pretty vague and don't tell user what to do next.